appengine-java11/quarkus-helloworld/pom.xml

```diff
@@ -29,8 +29,10 @@
   </parent>
 
   <properties>
+    <surefire-plugin.version>2.22.0</surefire-plugin.version>
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.source>11</maven.compiler.source>
+    <quarkus.version>0.13.1</quarkus.version>
```

Comment:
Why not https://github.com/quarkusio/quarkus/releases/tag/1.2.0.Final ?

Reply:
Thanks for catching this!

Reply:
Thanks... Did you deploy/access it for sanity check?

Reply:
Yes, good to go